.github/workflows/build-test.yml

```diff
@@ -725,18 +725,13 @@ jobs:
   clippy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-
-    # We don't expect to keep being clippy-clean on nightly Rust,
-    # so the "toolchain boilerplate" is deliberately omitted
-    # This just means we end up testing on the pinned stable even in the cronjob
-    # CI, which isn't that useful but specially omitting that task is more work than it's worth
-
-    # Clippy job > Install and run clippy steps
-    - name: Install clippy
-      run: rustup component add clippy
-
-    - uses: actions-rs/clippy-check@v1.0.7
+    - uses: actions/checkout@v3
+    - uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
```

Comment:
In two days we'll need to fix all 1.67 lints as well. Maybe set this to 1.66 for now and do 1.67 in a separate PR.

Reply:
oh, didn't notice that and did 1.67 already.